src/test/javascript/spec/component/exam/manage/student-exams/student-exam-detail-table-row.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { StudentExamDetailComponent } from 'app/exam/manage/student-exams/student-exam-detail.component';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Course } from 'app/entities/course.model';
+import { ArtemisDataTableModule } from 'app/shared/data-table/data-table.module';
+import { AlertComponent } from 'app/shared/alert/alert.component';
+import { ArtemisDurationFromSecondsPipe } from 'app/shared/pipes/artemis-duration-from-seconds.pipe';
```

Comment:
`'ArtemisDurationFromSecondsPipe' is defined but never used`

Reply:
Done